plugins/nevermind.py

```diff
@@ -0,0 +1,14 @@
+from errbot import BotPlugin, re_botcmd
+
+class Nevermind(BotPlugin):
+    """
+    Doesn't mind
+    """
+
+    @staticmethod
+    def nm_msg():
+        return "I'm sorry :("    
```

Comment:
@SanketDG yeah, I think we have to decide whether we just want to test the helper functions or do a full fledged unit as well as integration test with errbot as well.